changelogs/changelog_X_XX_X.md

```diff
@@ -1,7 +1,6 @@
 # v1.xx.x - yyyy-mm-dd
```

Comment:
Do we even need this file now? Can't this be part of `changelog.md`?

Reply:
we need it to simplify creating a new changelog, however, I've now merged changelogs/readme.md into changelog.md since 1 of them became redundant